Pokopom/General.h

```diff
@@ -30,7 +30,7 @@
 	#include <X11/Xlib.h>
 
 	#ifdef __cplusplus
-	#define DllExport extern "C" __attribute__((stdcall))
+	#define DllExport extern "C" __attribute__((stdcall)) __attribute__((visibility ("default")))
```

Comment:
Agreed. I tried to do that originally but I wasn't sure if the visibility attribute needed to be enclosed in the same set of parentheses as the stdcall parameter. This looks like it works though, thanks!